libindy/src/api/mod.rs

```diff
@@ -225,3 +225,16 @@ pub enum ErrorCode
     // Extra funds on inputs
     PaymentExtraFundsError = 705
 }
+
+#[no_mangle]
+pub extern fn indy_set_crypto_thread_pool_size(size: usize) -> ErrorCode {
```

Comment:
I believe it should be called indy_init() and require a json with crypto_thread_pool_size fields as an input.